Resources/Textures/Clothing/Head/Soft/bizzaresoft.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from civstation at commit https://github.com/Civ13/Civ13/commit/ec52cbb95d59b717d4d8c480b35ac133e5b58088#diff-fba188fb2db5d16e5d41985147336a8f96085f761b903e016fffd869b63e497d",
```

Comment:
Using the diff for it seems like a bad idea.

Also do you know if the asset license is actually CC-BY-SA-3.0? I can't find it on their repo.

Reply:
Okay Ike told me it should be NC anyway.